kafka_consumer/ci/kafka_consumer.rake

```diff
@@ -4,29 +4,42 @@ def kafka_consumer_version
   ENV['FLAVOR_VERSION'] || 'latest'
 end
 
+def kafka_consumer_options
+  ENV['FLAVOR_OPTIONS'] || 'zookeeper'
+end
+
+def kafka_topics
+  ENV['KAFKA_TOPICS'] || 'marvel:2:1,dc:2:1'
+end
+
+def zookeeper_version
+  ENV['ZOOKEEPER_VERSION'] || '3.4.9'
+end
+
 namespace :ci do
   namespace :kafka_consumer do |flavor|
     task before_install: ['ci:common:before_install']
 
     task :install do
       Rake::Task['ci:common:install'].invoke('kafka_consumer')
-      sh %(docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka_consumer/ci/resources/docker-compose-single-broker.yml up -d)
+      sh %(EXTERNAL_PORT=9092 EXTERNAL_JMX_PORT=9999 CONSUMER_OFFSET_STORAGE=#{kafka_consumer_options} KAFKA_TOPICS=#{kafka_topics} \
+           ZOOKEEPER_VERSION=#{zookeeper_version} \
+           docker-compose -f #{ENV['TRAVIS_BUILD_DIR']}/kafka_consumer/ci/resources/docker-compose-single-broker.yml up -d)
       Wait.for 2181
       Wait.for 9092
-      wait_on_docker_logs('resources_kafka_1', 5, '[Kafka Server 1001], started')
-      sh %(docker run -d --name kafka_consumer -v /var/run/docker.sock:/var/run/docker.sock -e HOST_IP=172.17.0.1 \
-           -e ZK=172.17.0.1:2181 -i -t wurstmeister/kafka:#{kafka_version} /bin/bash -c '$KAFKA_HOME/bin/kafka-console-consumer.sh \
-           --topic=test --zookeeper=$ZK --consumer-property group.id=my_consumer ')
-      wait_on_docker_logs('resources_kafka_1', 30, 'Created log for partition [test,0]')
-      sh %(docker run -d --name kafka_producer -v /var/run/docker.sock:/var/run/docker.sock -e HOST_IP=172.17.0.1 \
-           -e ZK=172.17.0.1:2181 -i -t wurstmeister/kafka:#{kafka_version} /bin/bash -c '$KAFKA_HOME/bin/kafka-console-producer.sh \
-           --topic=test --broker-list=`broker-list.sh` <<< "boomshakalaka"')
+      wait_on_docker_logs('resources_kafka_1', 20, '[Kafka Server 1001], started')
```

Comment:
It would be better to set a Docker HealthCheck in the docker-compose than waiting for the logs. But that may be part of another PR since we do that for other integrations.